content-src/components/Header/Header.js

```diff
@@ -24,8 +24,7 @@ const Header = React.createClass({
         {props.userName && <span>
           {props.userName}
         </span>}
-        {props.userImage && <img alt=""
-          src={props.userImage} />}
+        {props.userImage && <img alt="" src={props.userImage} />}
```

Comment:
ESLint really didn't like having the `<img>` open on one line and and closing on another, and I couldn't figure out how it wanted me to position that closing `/>`. So I cheated and made it a single line since it was short.